dll/win32/advapi32/advapi32.spec

```diff
@@ -682,3 +682,5 @@
 682 stub WmiSetSingleItemW
 683 stub Wow64Win32ApiEntry
 684 stdcall WriteEncryptedFileRaw(ptr ptr ptr)
+685 stdcall RegLoadMUIStringW(ptr wstr wstr long ptr long wstr)
```

Comment:
This is not okay. Our imports must match Win2003.
If our desk.cpl and shell32.dll import these functions, they will no longer work on Win2003.

Reply:
Don't we have "advapi32_vista.dll" ?

Reply:
But shell32.dll even now have this 
remove_definitions(-D_WIN32_WINNT=0x502)
remove_definitions(-D_WIN32_WINNT=0x600)

As well as 60+ other DLLs. Some of them redefine this config to x601..x603. It assumed that thay can not work as x502, isn't it?

Reply:
Arghhh, these xxx_vista.dll even shitter than this)

Reply:
@ThFabba  What if i try to wrap this copy-paste into #ifdef < 0x600  ?

Reply:
Yes, moving it to advapi32_vista would work to de-duplicate the code. That dll can be copied to Win2003 so it doesn't break this method of testing.

Wrapping the duplicated copies in an ifdef seems acceptable, though I'm not sure what it achieves. If your goal is just to fix build with that define, I guess that works.